backend/app/api/v1/lantern_api.py

```
@@ -78,25 +79,24 @@ async def get_lanterns(
+        400: error_400,
+        404: error_404,
+        500: error_500
     }
 )
 async def get_lantern_detail(
-        lantern_id: str,
-        current_lantern_id: str = Query(...),
+        lantern_id: str = Path(
+            ...,
+            description="조회할 랜턴의 ID",
+            regex = r"^[가-힣a-zA-Z0-9]+-[0-9]{4}$"
```

Flake8 (7.2.0)

[error] unexpected spaces around keyword / parameter equals

(E251)